CONTRIBUTING.md

```diff
@@ -1,17 +1,17 @@
 # Node.js Website Contributing Guide
 
-Thank you for your interest in contributing to the Node.js Website. Before you proceed, briefly go through the following:
+Thank you for being so interested in contributing to the Node.js Website. Before you proceed, briefly go through the following:
```

Comment:
I suggest reverting this. It reads condesending or patronizing to my English as a first language ears.